blueprint/index.md

```diff
@@ -0,0 +1,164 @@
+* Node.js installed. Version used: 14.0.0. For more information, see [Node.js](https://nodejs.org/en/ "Opens the Node.js page") on the Node.js website.
+* AWS SAM CLI installed. Version used: 1.23.0. For more information, see [Install AWS SAM](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html "Opens the Install AWS SAM CLI page") on the AWS website.
+
+## Implementation steps
+
+* [Clone the repository containing the project files](#clone-the-repository-containing-the-project-files "Goes to the Clone the repository containing the project files section")
+* [Enable the Amazon EventBridge integration in your Genesys Cloud account](#enable-the-amazon-eventbridge-integration-in-your-genesys-cloud-account "Goes to the Enable the Amazon EventBridge integration in your Genesys Cloud account section")
+* [Configure your EventBridge software as a service (SaaS) integration](#configure-your-eventbridge-software-as-a-service-saas-integration "Goes to the Configure your EventBridge software as a service (SaaS) integration section")
+* [Edit the configuration files](#edit-the-configuration-files  "Goes to the Edit the configuration files section")
+* [Deploy the application](#deploy-the-application "Goes to the Deploy the application section")
+* [Delete an OAuth Client](#delete-an-oauth-client "Goes to the Delete an OAuth Client section")
+* [Running locally](#running-locally "Running locally")
+
+### Clone the repository containing the project files
+
+Clone the [aws-eventbridge-oauth-client-delete-blueprint](https://github.com/GenesysCloudBlueprints/aws-eventbridge-oauth-client-delete-blueprint "Opens the aws-eventbridge-oauth-client-delete-blueprint repository in GitHub") repository from GitHub.
+
+### Enable the Amazon EventBridge integration in your Genesys Cloud account
+
+This step is only necessary if the Amazon EventBridge integration is not yet enabled in your Genesys Cloud account.  
+
+Follow the steps in [About the Amazon EventBridge integration](https://help.mypurecloud.com/articles/about-the-amazon-eventbridge-integration/ "Opens the About the Amazon EventBridge integration on the Genesys Cloud Resource Centre").
+
+### Configure your EventBridge software as a service (SaaS) integration
+
+Configure your [EventBridge software as a service (SaaS)](https://console.aws.amazon.com/events/home?region=us-east-1#/partners) integration, and note the event source name (e.g., aws.partner/example.com/1234567890/test-event-source). Before proceeding, ensure that your event source is listed as Pending.
+
+### Edit the configuration files
+
+Edit `src/python/config.py` and `src/typescript/src/config.ts` and add a PagerDuty API token.  
+
+Feel free to remove the source code of either Lambda function and remove the references to it from `template.yml` if you don't wish to use it.
+
+### Deploy the application
+
+From the repo root, run the following command:
```

Comment:
A little explanation of these commands would be nice for readers that aren't familiar with sam. No need to redocument them, just something mentioning that the build command gathers the source and creates a zip file (at least that's what I think it does), and the deploy command does the processes to upload that file and create your resources in your AWS account. The latter is particularly important to note; readers will want to know when they are making something live and/or may incur costs.